### PR TITLE
8 new epsiode sql and r content and examples

### LINKED ISSUE #8: New epsiode: SQL and R
Create SQL lesson. Incorporate additional AMR related data from[ Fingertips](https://fingertips.phe.org.uk/), using R package.

Exercises would need to be created to support inspired content from: https://dept.stat.lsa.umich.edu/~jerrick/courses/stat506/08-sql.html

### PR DESCRIPTION
**Pull request by quirksahern**

### CODE REVIEW

File: episodes/27-sql-r.Rmd [hunk 1]
```diff
@@ -0,0 +1,448 @@
+---
+title: SQL and R
+teaching: 110
+exercises: 40
+source: Rmd
+---
+
```
Comment (milanmlft) [inline episodes/27-sql-r.Rmd:7]: Missing `questions` callout

File: episodes/27-sql-r.Rmd [hunk 2]
```diff
@@ -0,0 +1,448 @@
...
+- Explain what SQL is and why to use it.
+- Understand how SQL can be used to query databases
+- Write queries that SELECT data based on conditions, such as AND, OR, and NOT and use of the WHERE clause.
+- Use SQL functions like AVG in combination with clauses like Group By to aggregate values and make calculations on fields using SQL.
+- Understand how to link tables together via joins.
+
+::::::::::::::::::::::::::::::::::::::::::::::::::
+
+:::::::::::::::::::::::::::::::::::::::: content
```
Comment (milanmlft) [inline episodes/27-sql-r.Rmd]: `content` is not a valid sandpaper callout

See available callouts at [Introduction to The Carpentries Workbench: Workbench Component Guide](https://carpentries.github.io/sandpaper-docs/component-guide.html)

**Review by milanmlft — APPROVED** [review]
Looks good, just need to check the callout close tags as they seem to be mismatched (see the GHA workflow comments)

File: episodes/27-sql-r.Rmd (hunk 2 above)
Comment (quirksahern) [reply episodes/27-sql-r.Rmd]: I'd put content, where I would have put questions. Have fixed issue.

File: episodes/27-sql-r.Rmd (hunk 1 above)
Comment (quirksahern) [reply episodes/27-sql-r.Rmd:7]: questions added

**Comment by milanmlft** [comment]
@quirksahern there are still problems with the callout block tags in `27-sql-r.Rmd`

**Pull request closed without merging**